TUT/STA130F24_TUT06_Oct25.ipynb

```diff
@@ -0,0 +1,12137 @@
+    "fig.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e99b40a9",
+   "metadata": {},
+   "source": [
+    "## Feedback + Discussion of Assumptions [10 minutes]\n",
+    "\n",
+    "### (Use images for reference) \n",
+    "> ![](https://stats.libretexts.org/@api/deki/files/1599/imageedit_3_7796089347.png?revision=1)\n",
+    "<img src=\"https://sphweb.bumc.bu.edu/otlt/MPH-Modules/BS/R/R5_Correlation-Regression/Illustration%20of%20Simple%20Linear%20Regression.gif\" alt=\"\" style=\"width: 450px;\"/>\n",
+    "\n",
+    "\n",
+    "**The error terms follow the normal distribution, as assumption which can be verified by inspecting that the empirical distribution of the residuals appears to be bell shaped.**\n",
+    "\n",
+    "**The assumption of homoscedasticity is satsified iff the variance of the error terms is constant, or does not depend on the independent variable. (important distinction in graphs above, in the first graph, we observe what is known as _\"hetroscedasticity\"_)**\n",
+    "\n",
+    "**The linear form states that the change in the dependent variable must be proportional to the change in independent variable (consider the equation of simple linear regression).**\n",
+    "\n",
+    "**There is no relationship between $Y_i$ (the outcome variables) execpt for how they depend on $x_i$ (the independent variables) (known as the assumption of independence).**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "43185ef7",
+   "metadata": {},
+   "source": [
+    "## Demo [15 minutes]\n",
+    "\n",
+    "- Use ChatGPT to create a dataset having several variables related to a musical show\n",
+    "- Identify two sets of predictor (independent) and outcome (variables) with students and plot them.\n",
+    "(one of the independent variables should be a indicator variable which should be used to transition into next part of tutorial\n",
+    "- Demonstrate using smf how a trendline can be plotted and how we can use it to find estimates of the parameters of the linear regression model.\n",
+    "- Demonstrate how to display the summary of the smf model and highlight key areas **(Coefficients of Intercept and Slope, R-squared value, p value)**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a04a508",
+   "metadata": {},
+   "source": [
+    "## Indicator Variables [15 minutes]\n",
+    "\n",
+    "- Start with a discussion of indicator variables\n",
+    "### Brief description of Broadway dataset\n",
+    "- We are looking at a sample of weekly broadway musical data.\n",
+    "- The current variables we are looking at are the average ticket price of a broadway musical show and whether that show was a 'Hamilton' or not.\n",
+    "- The 'Hamilton' variable is what is referred as a indicator variable, representing presence or absence of some category.\n",
+    "- As will be shown below, indicator variables are very important in regression analysis."
```

Comment:
Indicator variables go in my Nov04 LEC 07 as part of my "group comparison" lecture which will serve to function as a review/progression for the return from reading week

Reply:
**So you'll need a new example to demonstrate SLR**

Reply:
the general objective of which is discussed through the flow of feedback comments above 
- it does look like your structure here essentially looks good -- it just needs to not use indicator variables and then demo how to handle both significant and insignificant coefficient situations, and should be an interesting good data set demonstration of this that illustrates the point well, especially that statistical significance is a tool to help us assess things when it might be unclear visually